coordinape/coordinape
# Add org members

This adds a new Add Members page to the org members area, accessible from a button on the members list, similarly to the circle members list. You can add members directly, via CSV, or with an invite link. (Guild is not supported yet.)

It modifies the circle invite link logic and landing page to support orgs as well.

## testing

- Verify that the button appears for you on the org members page, and the path `/organizations/:org_id/members/add` is accessible, iff you are an admin of any circle in the org.
- Add members using the form and CSV import.
- Visit the invite link while logged out and verify that you can see info about the org, plus a Connect Wallet button, which becomes a Join button once you connect a wallet. Clicking this button should add you to the org and redirect you to its main page.
